sharding/colocation.rst

```diff
@@ -90,12 +91,11 @@ Across all shards of the event table (Q2):
 
   SELECT page_id, count(*) AS count
   FROM event
-  WHERE page_id IN (ARRAY[...page IDs from first query...]) AND
-        tenant_id = 6 AND
-        payload->>'time' >= now() - interval '1 week'
+  WHERE page_id IN (…page IDs from first query…)
```

Comment:
This overall looks good, but I'm wondering if this section needs some rewrite now that we have broader SQL support in 7.2. For example, we could actually write this query as a subquery in 7.2, which sort of diminishes the point we're trying to make.